### PR TITLE
resumeGeneratorFormv3

### PR DESCRIPTION
**Pull request by Christopher-Tom**
hello, please look at this 👍 
You can put in the information or just spam next page and click submit 👍 
That is all 👍

### CODE REVIEW

**Review by IsitaB — COMMENTED** [review]
fix adminnavbar frontend tests

**Comment by qsaml** [comment]
is it just me, or does no pdf download?

some fields could be more intuitive. ones that say "type skills here" should have an additional specifier, maybe "separated by commas"

page itself looks really clean though

File: src/Components/Navbar/UserNavbar.js [hunk 1]
```diff
@@ -243,19 +258,30 @@ export default function UserNavBar(props) {
...
+                    </NavItem>
+                  );
+                })}
+                <UncontrolledDropdown nav inNavbar>
+                  <DropdownToggle id='navlink-text' nav caret>
                   Printing
-                </DropdownToggle>
-                <DropdownMenu right>
-                  <DropdownItem className='drp-item' href='/2DPrinting'>
+                  </DropdownToggle>
```
Comment (IsitaB) [inline src/Components/Navbar/UserNavbar.js:274]: I guess the insides didnt get indented. just `OPT+SHIFT+F` this file

File: src/Pages/Resume/Education.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {FormGroup, Input} from 'reactstrap';
+class Education extends React.Component{
+  render(){
+    if(this.props.currentStep !==2){
+      return null;
+    }
+    const nameStyle = {
```
Comment (IsitaB) [inline src/Pages/Resume/Education.js:8]: move it to your css file

File: src/Pages/Resume/Experience.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    this.props.handleChange(event, this.props.index, 'experience');
+  }
+  render() {
+    let btnVisible = (this.state.visibleButton) ?
+      'customBtnVisible' : 'customBtnHidden';
+
+    if (this.props.currentStep !== 3) {
+      return null;
+    }
+    const nameStyle = {
```
Comment (IsitaB) [inline src/Pages/Resume/Experience.js:30]: this is the same as the other nameStyle. move it to a css file. DRY

File: src/Pages/Resume/GeneralInformation.js [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  render(){
+    if(this.props.currentStep !==1){
+      return null;
+    }
+    const nameStyle = {
+      margin: 'auto',
+      width: '35%',
+      fontFamily: 'Popper',
+    };
+    return(
```
Comment (IsitaB) [inline src/Pages/Resume/GeneralInformation.js:14]: I'd say use arrays and map functions. these form groups all look similar

**Review by IsitaB — CHANGES_REQUESTED** [review]
it works!! 🔥🔥🔥
Screen Shot 2020-09-08 at 8 15 52 PM
fix this. you use "class" but you should be using "className" instead

you need padding bottom for your page

rebase

**Comment by jerrylee17** [comment]
See #661

**Pull request closed without merging**